ops/charm.py

```diff
@@ -69,21 +70,20 @@
         'multiple': _MultipleRange
     })
 
-    _ResourceMetaDict = TypedDict(
-        '_ResourceMetaDict', {
-            'type': Required[str],
-            'filename': str,
-            'description': str},
-        total=False)
+    class _ResourceMetaDict(TypedDict, total=False):
+        type: Required[str]
+        filename: str
+        description: str
+
+    class _PayloadMetaDict(TypedDict):
+        type: str
 
-    _PayloadMetaDict = TypedDict('_PayloadMetaDict', {'type': str})
+    class _MountDict(TypedDict, total=False):
+        storage: Required[str]
+        location: str
 
-    _MountDict = TypedDict(
-        '_MountDict', {'storage': Required[str],
-                       'location': str},
-        total=False)
-    _ContainerMetaDict = TypedDict(
-        '_ContainerMetaDict', {'mounts': List[_MountDict]})
+    class _ContainerMetaDict(TypedDict):
+        mounts: List[_MountDict]
 
     _CharmMetaDict = TypedDict(
```

Comment:
This should also be a class, vs a TypedDict instantiation

Reply:
Not quite sure what you mean here?